discretize/View.py

```diff
@@ -778,7 +778,10 @@ def plot_3d_slicer(self, v, xslice=None, yslice=None, zslice=None,
         fig.canvas.mpl_connect('scroll_event', tracker.onscroll)
 
         # Show figure
-        plt.show()
+        if showIt:
+            plt.show()
+
+        return fig
```

Comment:
We might want to think about the `return fig`. It has the `fig` parameter in the call, so you can provide an exiting figure, in which case you don't want/need it to be returned. Is it always required?

But anyhow, as long as it is streamlined with `discretize` then I'm fine with it.

Reply:
The main motivation for the return here would be if someone didn't provide it as an input, but instead let the figure handle be created by the function. Then if you want to add labels and such after-the-fact, you can access the figure handle

Reply:
Yeah, I think this is used a lot in modifying figures after the fact. Returning the thing you created seems like good practice.